docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -9,6 +9,22 @@ If your problem isn’t described here, please review open issues in the followi
 
 Have a question? Contact us in the https://discuss.elastic.co/[discuss forum].
 
+[float]
+[[rule-fails-to-decrypt-api-key]]
+=== Failing to decrypt apiKey
+
+*Problem*:
+
+The rule fails to execute and has an `Unable to decrypt attribute "apiKey"` error.
+
+*Solution*:
+
+This error happens when the `xpack.encryptedSavedObjects.encryptionKey` value changed since creating the rule. Depending on the scenario, there are different ways to solve this problem:
+
+* Suppose the value in `xpack.encryptedSavedObjects.encryptionKey` was manually changed, and the previous encryption key is still known. In that case, any previous encryption key needs to be part of the <decryption only key, encryption-key-rotation>.
```

Comment:
Is this meant to be a link? I think you need double brackets to linkify it.

Reply:
It should be double brackets, and the contents is in the wrong order: 

<<encryption-key-rotation, decryption only key>>

But the text "decryption only key" doesn't make sense to me.

Reply:
Yeah, it was meant to be a link, though I changed to go to the kibana yml settings docs: https://www.elastic.co/guide/en/kibana/current/security-settings-kb.html#xpack-encryptedSavedObjects-keyRotation-decryptionOnlyKeys